blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_3a.py

```diff
@@ -0,0 +1,51 @@
+"""Formula 5.3a from NEN-EN 1992-1-1+C2:2011: Chapter 5 - Structural Analysis."""
+# pylint: disable=arguments-differ
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import DIMENSIONLESS, KN
+from blueprints.validations import raise_if_negative
+
+
+class Form5Dot3ATransverseForceUnbracedMembers(Formula):
+    """Class representing formula 5.3a for the calculation of the transverse force for unbraced members, Hi.
```

Comment:
I value your welcoming 'Hello' but I think it is a bit confusing ;-) We have to decide how to handle subscripts

Reply:
Johan has also brought this up a few times. But don't worry, I'm aware of this issue and I'll address it as soon as possible.